synapse/config/server.py

```diff
@@ -136,21 +136,6 @@ def read_config(self, config):
                 "type": "manhole",
             })
 
-        metrics_port = config.get("metrics_port")
-        if metrics_port:
-            self.listeners.append({
-                "port": metrics_port,
-                "bind_addresses": [config.get("metrics_bind_host", "127.0.0.1")],
-                "tls": False,
-                "type": "http",
-                "resources": [
-                    {
-                        "names": ["metrics"],
-                        "compress": False,
-                    },
-                ]
-            })
```

Comment:
Do we not need to keep this for backwards compat for people using `metrics_port`?